### DISCUSSION TITLE
Is anyone interested in comma separated currency?

### DISCUSSION THREAD

**Discussion by zbjdonald** [directus/directus#15562] [Feature Requests]
![image](https://user-images.githubusercontent.com/5664723/190044331-ee83e022-9a2e-4b42-b683-a0c5c903a2d3.png)

**Comment by benhaynes** [comment]
Would love a mask/format option for all number fields! Probably on the base component, and only displays when not in "edit mode".

**Reply by raflymln** [reply]
I'd love to see this, and it'd be great if it can be implemented on edit mode, anyways, there's a plugin that allows mask on input, maybe you could consider that extension to be implemented inside the official package.

https://github.com/dimitrov-adrian/directus-extension-masked-interface

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
